.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,54 @@
+<!-- Please refer to our contributing documentation for any questions on submitting a pull request, or let us know here if you need any help: https://github.com/ionic-team/stencil/blob/master/.github/CONTRIBUTING.md -->
+
+## Pull request checklist
+
+Please check if your PR fulfills the following requirements:
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
+- [ ] Build (`npm run build`) was run locally and any changes were pushed
+- [ ] Unit tests (`npm test`) were run locally and passed
+- [ ] E2E Tests (`npm run test.karma.prod`) were run locally and passed
+- [ ] Prettier (`npm run format`) was run locally and passed
```

Comment:
Prettier is currently `npm run prettier` (but I would not be opposed to making it `npm run format`):
https://github.com/ionic-team/stencil/blob/252415a5cc94cad9ea0a5eddb73bae32d1845c96/package.json#L39

Reply:
Bah, that's my fault. In a former life I named the script 'format'. Let me change that realllll quick